co2calculator/distances.py

```diff
@@ -287,36 +299,6 @@ def geocoding_train_stations(loc_dict):
     return res_station_name, res_country, coords
 
 
-def is_valid_geocoding_dict(geocoding_dict):
```

Comment:
I gues this validation could also be done using a pydantic object right?

Reply:
Yes, this is why the function is deleted  - it is now done with the pydantic model:
https://github.com/pledge4future/co2calculator/blob/ff86e53720e6b096356601285789982977cbfd69/co2calculator/distances.py#L34-L42

It makes sure that `locality` and `country` have to be provided and that no other parameters are given (with the `Extra.forbid`)

Reply:
ah yes of course 😅